<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="12141" started-at="2022-03-12T07:11:33Z" finished-at="2022-03-12T07:11:45Z">
    <groups>
    </groups>
    <test name="Google Test" duration-ms="7155" started-at="2022-03-12T07:11:33Z" finished-at="2022-03-12T07:11:40Z">
      <class name="sample1.Test1">
        <test-method status="FAIL" signature="validateGoogle()[pri:0, instance:sample1.Test1@64616ca2]" name="validateGoogle" duration-ms="7118" started-at="2022-03-12T12:41:33Z" finished-at="2022-03-12T12:41:40Z">
          <exception class="java.lang.ArithmeticException">
            <message>
              <![CDATA[/ by zero]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.ArithmeticException: / by zeroat sample1.Test1.validateGoogle(Test1.java:37)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.ArithmeticException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateGoogle -->
      </class> <!-- sample1.Test1 -->
    </test> <!-- Google Test -->
    <test name="Bing Test" duration-ms="4967" started-at="2022-03-12T07:11:40Z" finished-at="2022-03-12T07:11:45Z">
      <class name="sample2.Test2">
        <test-method status="PASS" signature="validateGoogle()[pri:0, instance:sample2.Test2@366e2eef]" name="validateGoogle" duration-ms="4966" started-at="2022-03-12T12:41:40Z" finished-at="2022-03-12T12:41:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateGoogle -->
      </class> <!-- sample2.Test2 -->
    </test> <!-- Bing Test -->
  </suite> <!-- Suite -->
</testng-results>
